entrypoint.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+if [ "$1" == "web" ]
+then
+  gunicorn --error-logfile - -c /home/vcap/app/gunicorn_config.py application
```

Comment:
This should be started with exec

Reply:
Thanks, this is now updated (and the other place too)